documentation/src/main/asciidoc/topics/rest_api_v2.adoc

```diff
@@ -406,6 +406,201 @@ To obtain a list of caches available in a {brandname} cluster, invoke a `GET` re
+----
+GET /v2/caches/{cacheName}/x-site/backups/{backupName}?action=take-offline
+----
+
+==== Bringing a backup site online
+
+To take a backup site `backupName` online, for the cache `cacheName`, execute a `GET` request:
+
+[source,options="nowrap",subs=attributes+]
+----
+GET /v2/caches/{cacheName}/x-site/backups/{backupName}?action=bring-online
+----
+
+==== Starting a state push to a backup site
+
+To start pushing state of a cache `cacheName` to a remote backup site `backupName`, execute a `GET` request:
+
+[source,options="nowrap",subs=attributes+]
+----
+GET /v2/caches/{cacheName}/x-site/backups/{backupName}?action=start-push-state
+----
+
+==== Cancelling an ongoing state push to a backup site
+
+To cancel a state push of the cache `cacheName` to a remote backup site `backupName`, execute a `GET` request:
+
+[source,options="nowrap",subs=attributes+]
+----
+GET /v2/caches/{cacheName}/x-site/backups/{site}?action=cancel-push-state
+----
+
+==== Getting the status of a state push
+
+To obtain the status of an ongoing state push of cache `cacheName` to backup `backupName` execute a `GET` request:
+
+[source,options="nowrap",subs=attributes+]
+----
+GET /v2/caches/{cacheName}/x-site/backups/{backupName}?action=cancel-push-state
+----
+
+The response will be a JSON document with each destination site name and the state transfer status. Example:
+
+[source,json,options="nowrap",subs=attributes+]
+----
+{
+   "NYC":"CANCELED",
+   "LON":"OK"
+}
+----
+
+The possible statuses are `SENDING`, `OK`, `ERROR` and `CANCELLING`
+
+==== Tuning the take offline parameters of a remote site
+
+A remote site can be automatically marked as offline in case some conditions are met.
+To check the configured parameters, execute a `GET` request:
+
+[source,options="nowrap",subs=attributes+]
+----
+GET /v2/caches/{cacheName}/x-site/backups/{backupName}/take-offline-config
+----
+
+The response include two fields, `afterFailures` and `minTimeToWait`:
+
+[source,json,options="nowrap",subs=attributes+]
+----
+{
+  "afterFailures": 2,
+  "minTimeToWait": 1000
+}
+----
+
+To change those parameters, execute a PUT request:
+
+[source,options="nowrap",subs=attributes+]
+----
+PUT /v2/caches/{cacheName}/x-site/backups/{backupName}/take-offline-config
+----
+
+with a body containing the new values, e.g.:
+
+[source,json,options="nowrap",subs=attributes+]
+----
+{
+  "afterFailures": 4,
+  "minTimeToWait": 5000
+}
+----
+
+==== Cancelling the receiving state on a site
+
+The main use for this method is when the link between the sites is broken and the receiver site keeps it state transfer state forever.
+
+To set the cluster to normal state in the scope of `cacheName` for state pushed from site `pushingSiteName` execute a `GET` request:
+
+[source,options="nowrap",subs=attributes+]
+----
+GET /v2/caches/{cacheName}/x-site/backups/{pushingSiteName}?action=cancel-receive-state
+----
+
+The response include two fields, `afterFailures` and `minTimeToWait`:
```

Comment:
I don't think the `cancel-receive-state` has this response :)

Reply:
Sorry, it was a copy and paste...